static/js/types/limits.ts

```diff
@@ -0,0 +1,42 @@
+export enum MEM_UNITS {
+  B = "B",
+  KB = "kB",
+  MB = "MB",
+  GB = "GB",
+  TB = "TB",
+  PB = "PB",
+  EB = "EB",
+  KIB = "KiB",
+  MIB = "MiB",
+  GIB = "GiB",
+  TIB = "TiB",
+  PIB = "PiB",
+  EIB = "EiB",
+}
+
+export enum MEM_LIMIT_TYPE {
+  FIXED = 0,
+  PERCENT = 1,
+}
+
+export interface MemoryLimit {
+  value: number | null;
+  unit: MEM_UNITS | "%";
+  selectedType: MEM_LIMIT_TYPE;
+}
+
+export enum CPU_LIMIT_TYPE {
+  DYNAMIC = 0,
+  FIXED_RANGE = 1,
+  FIXED_SET = 2,
+}
+
+export interface CpuLimit {
+  dynamicValue: number | null;
```

Comment:
I think those (and MemoryLimit above) should be optional, as in `dynamicValue?: number`.